src/components/AgeVerifier.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import DatePicker from "react-datepicker";
+import { Alert } from "@mui/material";
+import "react-datepicker/dist/react-datepicker.css";
+import "./AgeVerifier.css";
+
+const AgeVerifier = () => {
+  const [underageAlert, setUnderAgeAlert] = useState(false);
+
+  const handleCloseAlert = () => {
+    setUnderAgeAlert(false);
+  };
+
+  const DPChange = (input) => {
```

Comment:
What is the purpose of this function?

Reply:
i can't find it where it is being used?

Reply:
I made this function and DPSubmit to check the variation in DatePicker's behaviour with onChange and OnSubmit functions, I removed this during code cleanup, though that commit wasn't pushed. it's removed now